contrib/node/src/python/pants/contrib/node/subsystems/node_distribution.py

```diff
@@ -36,7 +36,7 @@ def register_options(cls, register):
       register('--supportdir', advanced=True, default='bin/node',
                help='Find the Node distributions under this dir.  Used as part of the path to '
                     'lookup the distribution with --binary-util-baseurls and --pants-bootstrapdir')
-      register('--version', advanced=True, default='6.9.1',
+      register('--version', advanced=True, default='8.6.0',
```

Comment:
Are you ready to bump this as well, or is this from your debugging?

Reply:
Done.